common.sh

```diff
@@ -509,3 +509,21 @@ if [[ -n "$MIRROR_IMAGES" || -z "${IP_STACK:-}" || "$IP_STACK" = "v6" || "$IP_ST
 fi
 
 export AGENT_TEST_CASES=${AGENT_TEST_CASES:-}
+
+#
+# Manage libvirtd services based on OS
+#
+manage_libvirtd() {
```

Comment:
Unless not needed elsewhere in future, it seems to me that the function defintion could be better placed in step 02, since it's the one that mostly takes care of the networking setup, rather than in `common.sh`, which is usually imported by all the other steps.

Reply:
In fact, I'm wondering if we couldn't import metal3-dev-env `lib/common.sh` ? It seems most of the things are duplicated there, with some additions.
I hesitated putting this method in the 02, but since it's pretty generic, thought it would be best in common.sh to get it properly exposed in case of future iteration/additions.. I can of course move it.

Reply:
'k - I'll push an update tomorrow then. Defining a method in the 02 is still OK, so that it may be easier to move it around?